test/wrapping-re.t/run.t

```diff
@@ -2174,40 +2176,46 @@ Format wrapping in .re files
    *     );
    *
    */
-  let df_locallyAbstractFuncAnnotated:
-    type a b.
-      (a, b) =>
-      (inputEchoRecord(a), inputEchoRecord(b)) =
-    (input: a, input2: b) => (
-      {inputIs: input},
-      {inputIs: input2},
-    );
+  let df_locallyAbstractFuncAnnotated
+      : type a b.
+          (a, b) =>
+          (
+            inputEchoRecord(a),
+            inputEchoRecord(b),
+          ) =
+      (input: a, input2: b) => (
+    {inputIs: input},
+    {inputIs: input2},
+  );
   
   /**
    * This case shows why inferring what was originally sugar type a b . blahblah
    * is not so trivial. We have to take the last Pexp_constraint type, varify the
    * constructors, then check if the result is equal to the first
    * Ppat_constraint. In this case, they're not equal!
    */
-  let df_locallyAbstractFuncAnnotated: 'figureMeOut =
-    (type a, type b) => (
-      (input: a, input2: b) => (
-        {inputIs: input},
-        {inputIs: input2},
-      ):
-        (a, b) =>
-        (inputEchoRecord(a), inputEchoRecord(b))
+  let df_locallyAbstractFuncAnnotated
+      : 'figureMeOut =
+      (type a, type b)
+      : (
+          (a, b) =>
+          (
+            inputEchoRecord(a),
+            inputEchoRecord(b),
+          )
+        ) =>
+    (input: a, input2: b) => (
+      {inputIs: input},
+      {inputIs: input2},
```

Comment:
This is a weird printing

Reply:
I think it's a pretty big edge case-- no one really writes code like this.

I'm tempted to be reactive here since everything else looks pretty good.

Please feel free to report weird printing when you upgrade internally and I'd be happy to fix those cases.